client/src/styles/Geocoder.css

```diff
@@ -0,0 +1,79 @@
+.leaflet-control-geocoder {
+  top: 1rem;
+  left: -1rem;
+  z-index: 1000;
+  background-color: var(--fourth-color);
+  padding: 0.2rem;
+  border-radius: 1px solid black;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0, 2);
+  width: 80%;
+  display: flex;
+  justify-content: center;
+}
+
+.leaflet-control-geocoder-icon {
+  background-color: var(--third-color);
+  color: var(--fourth-color);
+  padding: 1.5rem;
+  border-radius: 0.5rem;
+  position: absolute;
+  right: 0;
+  top: 50%;
+  right: 10%;
+  transform: translateY(-50%);
+}
+
+.leaflet-control-geocoder-form {
+  color: whitesmoke;
```

Comment:
Where does "whitesmoke" comes from ?

Reply:
You are right, I pushed this PR again, hence the message on discord, an update to the PR has been made.